dashboard/app/views/levels/instructions.haml

```diff
@@ -0,0 +1,26 @@
+-#
+  This is a view for rendering a summary of all of the levels/instructions in a given
+  script, that should only be visible in levelbuilder mode
+
+- stages = local_assigns[:stages]
+
+- stages.each_with_index do |stage, index|
+  %h1{:style => "margin-top: 30px"}= "Stage #{(index + 1)}: #{stage.name}"
+  - stage.script_levels.each do |script_level|
+    - level = script_level.level
+    %div{:style => "border: 1px dashed black; padding: 10px"}
+      = link_to "Level #{script_level.position}: #{script_level.level.name}", build_script_level_path(script_level)
+      %div= "type: #{script_level.level.class}"
+      - levels = level.is_a?(LevelGroup) ? level.levels : [level]
+      - levels.each do |level|
+        - if level.class == External
```

Comment:
I wonder a little bit if that is over-engineering. This instructions aggregation is meant to be a quick and dirty feature that is a bit of an edge case. I kind of like that I can just go to instructions.haml and get a good sense of what everything looks like without having to crawl through a bunch of other files.

Willing to be convinced otherwise if you feel strongly about this.